parachain/pallets/incentivized-channel/src/outbound/mod.rs

```diff
@@ -153,6 +164,8 @@ impl<T: Config> Module<T> {
 		let message_count = messages.len();
 		T::WeightInfo::on_initialize(
 			message_count as u32,
+			// We overestimate message payload size rather than underestimate.
+			// So add 1 here to account for integer division truncation.
```

Comment:
Adding this in response to https://github.com/Snowfork/polkadot-ethereum/pull/342#discussion_r616110876